src/smartMirrorProject/noteWidget/static/noteWidget/style.css

```diff
@@ -0,0 +1,23 @@
+.notes-widget {
+    font-family: Arial, sans-serif;
+    background-color: #f9f9f9;
+    border: 1px solid #ddd;
+    border-radius: 8px;
+    padding: 20px;
+    max-width: 400px;
+    margin: 20px auto;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    max-height: 200px;
+    overflow-y: auto;
+}
```

Comment:
In design verbeteren is dit al aangepast, maar dingen als font-family, background-color, border-radius, padding, max-width en max-height, box-shadow en overflow-y zijn daar al centraal geregeld voor alle widgets. Weet niet of het handig is om dit zo mee te nemen en later dit recht te trekken of het gelijk nu goed te doen.

Reply:
Kan dit dan ff aanpassen. Zijn dit precies de regels die ik moet verwijderen?